lib/kbsecret/cli/kbsecret-cp

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "kbsecret"
+include KBSecret
+
+cmd = CLI.create do |c|
+  c.slop do |o|
+    o.banner = <<~HELP
+      Usage:
+        kbsecret cp <src-sess> <dst-sess> [options] <record [record ...]>
+    HELP
+    o.bool "-m", "--move", "delete the record after copying"
+  end
+
+  c.dreck do
+    string :src_sess
+    string :dst_sess
+    list :string, :labels
+  end
+end
+
+src_sess = Session.new label: cmd.args[:src_sess]
+dst_sess = Session.new label: cmd.args[:dst_sess]
+
+selected_records = src_sess.records.select { |r| cmd.args[:labels].include?(r.label) }
+cmd.die "No such record(s)." if selected_records.empty?
+
+selected_records.each do |record|
+  begin
```

Comment:
This command should fail if any exceptions are raised, so remove this `begin...rescue` block.

Reply:
Got it